hack/run-functests.sh

```diff
@@ -58,10 +67,10 @@ if [ "$TESTS_IN_CONTAINER" == "true" ]; then
   -v $(pwd)/_cache/:/kubeconfig:Z \
   -v $TESTS_REPORTS_PATH:/reports:Z \
   ${env_vars} \
```

Comment:
should you add the `TEST_SUITES` also here?

Reply:
it won't hurt.
I expect the three suites to run consistently when somebody runs `make functests`, but I can add that in case it's useful in the future (plus, we would have a consistent api outside)

Reply:
Just noticed I mis-understood you comment. Yes, you are right.
Just fixed it